src/main/java/dk/cngroup/lentils/controller/ProgressController.java

```diff
@@ -89,19 +92,18 @@ public String viewHintsForCypher(final @PathVariable("cypherId") Long cypherId,
         return HINT_LIST;
     }
 
-    @GetMapping(value = "/viewTakenHints/{cypherId}")
-    public String viewHintsForCypherByTeam(final @PathVariable("cypherId") Long cypherId,
-                                           final @RequestParam("teamId") Long teamId,
-                                           final @RequestParam("hintId") Long hintId,
-                                           final Model model) {
-        Cypher cypher = cypherService.getCypher(cypherId);
+    @PostMapping("/getHintOfCypher")
```

Comment:
Mozna bych to prejmenoval na "/takeHintOfCypher" protoze get invokuje, ze to je get request.

Reply:
Přejmenováno.